src/rules/whitespaceRule.ts

```diff
@@ -296,6 +296,10 @@ function walk(ctx: Lint.WalkContext<Options>) {
                 }
                 break;
             case ts.SyntaxKind.ImportKeyword:
+                if ((ts.SyntaxKind as any).ImportType !== undefined && parent.kind === (ts.SyntaxKind as any).ImportType) {
```

Comment:
Hmm, I don't like having to cast as any. What is the issue here - is it that `ts.SyntaxKind.ImportType` is a ts2.9+ thing?

Reply:
Yup.

Reply:
@giladgray @suchanlee @JKillian any ideas on how to work around this?

Reply:
Let's define a shared `isImportKeyword` in `src/utils.ts` similar to existing helper functions from tsutils.